/cext.rb:19:in `callback': Ruby callback error (RuntimeError)
	from PWD/lib/truffle/truffle/cext.rb:LINE:in `__send__'
	from PWD/lib/truffle/truffle/cext.rb:LINE:in `rb_funcall'
	from PWD/lib/truffle/truffle/cext.rb:LINE:in `rb_funcallv'
	from PWD/src/main/c/cext/ruby.c:LINE:in `rb_funcallv'
/cext.rb:23:in `ruby_call'
/cext.rb:27:in `top'
/cext.rb:31:in `block in <main>'
/backtraces.rb:17:in `check'
/cext.rb:30:in `<main>'
